go.mod

```diff
@@ -3,10 +3,13 @@ module sigs.k8s.io/scheduler-plugins
 go 1.15
 
 require (
-	github.com/google/go-cmp v0.4.0
+	github.com/francoispqt/gojay v1.2.13
```

Comment:
It is just a fast JSON parser https://github.com/francoispqt/gojay#decode, to avoid decode/encode bottlenecks. Let me know if you still want me to use native.

Reply:
Thank you for pointing this out, I was able to remove the `replace` hacks.